src/EventListener/AbstractCalloutDoctrineEventSubscriber.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusCalloutPlugin\EventListener;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Event\PostFlushEventArgs;
+use Setono\SyliusCalloutPlugin\Message\Command\AssignEligibleCalloutsToProduct;
+use Setono\SyliusCalloutPlugin\Message\Command\AssignEligibleCalloutsToProducts;
+use Setono\SyliusCalloutPlugin\Model\CalloutInterface;
+use Setono\SyliusCalloutPlugin\Model\ProductInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+abstract class AbstractCalloutDoctrineEventSubscriber
+{
+    /** @var EntityManager */
+    protected $calloutManager;
+
+    /** @var MessageBusInterface */
+    protected $commandBus;
+
+    /** @var bool */
+    protected $manualTriggering;
+
+    /** @var ArrayCollection|CalloutInterface[] */
+    private $calloutsToUpdate;
+
+    /** @var ArrayCollection|ProductInterface[] */
+    private $productsToAssign;
+
+    public function __construct(
+        EntityManager $calloutManager,
+        MessageBusInterface $commandBus,
+        bool $manualTriggering = false
```

Comment:
Is there a reason for this default value? I suppose you are injecting the `setono_sylius_callout.manual_triggering` parameter value when you define the services?